node.go

```diff
@@ -132,18 +113,31 @@ func (n *Node) Start() error {
 func (n *Node) pollReplyLoop() {
 	var timer time.Ticker
 
+	// create an ArtPollReply packet to send out in response to an ArtPoll packet
+	p := ArtPollReplyFromConfig(n.Config)
```

Comment:
Should this code https://github.com/jsimonetti/go-artnet/blob/08126d3107c76cb05585d4eaab98c17a84720779/controller.go#L136-L161 left unchanged? Maybe you could simplify the code by sending a trigger to `n.pollCh` in controller to let the underlying node emit his own reply instread of duplicating the block, what do you think?

Like:
```go
// send ArtPollPacket
c.cNode.sendCh <- netPayload{
	address: broadcastAddr,
	data:    b,
}
c.cNode.pollCh <- packet.ArtPollPacket{}
```

Reply:
This makes sense :)
Will do so.